src/theory/quantifiers/sygus/enum_value_manager.cpp

```diff
@@ -34,12 +34,14 @@ namespace theory {
 namespace quantifiers {
 
 EnumValueManager::EnumValueManager(Node e,
+                                   Env& env,
```

Comment:
I have a slight preference for always passing `Env` as the first argument. What do you think about that?

Reply:
I think it could go either way, I did it this way so that `env` directly proceeds `qs`. Perhaps `e` should be all the way towards the end.